### PR TITLE
Small changes

### PR DESCRIPTION
**Pull request by jrchatruc**

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -83,9 +82,9 @@ pub type FunctionKeys = (ProvingKey, VerifyingKey);
 pub fn execute_function(
     function: &Function<Testnet3>,
     user_inputs: &[UserInputValueType],
-    rng: &mut StdRng,
 ) -> Result<(SimpleFunctionVariables, MarlinProof)> {
-    let universal_srs = simpleworks::marlin::generate_universal_srs(rng)?;
+    let mut rng = simpleworks::marlin::generate_rand();
```
Comment (ilitteri) [inline src/lib.rs:86]:
```suggestion
    let rng = &mut simpleworks::marlin::generate_rand();
```
will let you remove `&mut` when passing as a parameter.

File: src/lib.rs [hunk 2]
```diff
@@ -184,8 +183,8 @@ pub fn verify_proof(
     verifying_key: VerifyingKey,
     public_inputs: &[UserInputValueType],
     proof: &MarlinProof,
-    rng: &mut StdRng,
 ) -> Result<bool> {
+    let rng = &mut simpleworks::marlin::generate_rand();
```
Comment (ilitteri) [inline src/lib.rs:187]: If it is only used as a parameter for `simpleworks::marlin::verify_proof` we could send `&mut simpleworks::marlin::generate_rand()` directly as a parameter and remove this line.

File: src/jaleo/mod.rs [hunk 3]
```diff
@@ -34,15 +38,32 @@ pub type ViewKey = snarkvm::prelude::ViewKey<Testnet3>;
...
+
+pub fn serialize_proof(proof: MarlinProof) -> Result<Vec<u8>> {
+    let mut bytes_proof = Vec::new();
+    proof
+        .serialize(&mut bytes_proof)
+        .map_err(|e| anyhow!("Error serializing proof: {e:?}"))?;
+
+    Ok(bytes_proof)
+}
+
```
[the remark this answers is not in the document]
Comment (jrchatruc) [reply src/jaleo/mod.rs:66]: Addressed here [e7f32d6](https://github.com/lambdaclass/VMtropy/pull/71/commits/e7f32d656889da971f22c26fb146382a2efb7b98)

File: src/lib.rs (hunk 2 above)
Comment (jrchatruc) [reply src/lib.rs]: Addressed here [e7f32d6](https://github.com/lambdaclass/VMtropy/pull/71/commits/e7f32d656889da971f22c26fb146382a2efb7b98)

File: src/jaleo/mod.rs (hunk 3 above)
[the remark this answers is not in the document]
Comment (jrchatruc) [reply src/jaleo/mod.rs]: Addressed here [71dfa36](https://github.com/lambdaclass/VMtropy/pull/71/commits/71dfa36700bf7c10465cff67b855699d986bb1cd)

**Review by ilitteri — APPROVED** [review]
LGTM

**Pull request merged**